### ISSUE TITLE
Reconcile Satlas-specific constants with Varnish2 

### ISSUE THREAD

**Issue by arnavic** [allenai/varnish-mui#136]
some specific examples...
- we have a font size of 17px 
- we have a border radius of 7px

best approach here would probably be to get fixed values from @aaronsarnat and then agree upon / implement the ones we want integrated into Varnish2

**Comment by aaronsarnat** [comment]
@arnavic asset is here: https://drive.google.com/file/d/1UVfhyl9ILTObIm9gzURIc4PfHD1roo4g/view?usp=drive_link

Will defer to PRIOR and/or Carissa for description text.

**Comment by aaronsarnat** [comment]
Agreed, we need a new design system. The one we have currently is no longer serving our needs and is thus introducing more and more design debt with each product that leverages vanilla Varnish.

Varnish is great, and I have no qualms with the concept or architecture, but it is currently very prescriptive about a legacy design system which is going to continue to be a pain point until we can invest in a new one. No ETA on starting that effort yet, as I am yet to get buy-in from management.

In the meantime, our options are:

1. Emphasize developer experience, use vanilla Varnish on all new projects and introduce more design debt
2. Emphasize aesthetics, go with more custom theming when needed and introduce more technical debt
3. Some combination which introduces both design and tech debt

If I can get buy-in to invest in a new design system then the result of that effort would be new values for Varnish, and then vanilla Varnish would be the path of least resistance for both Design and Eng. That'd be the goal anyway.

Unfortunately, I suspect we'll be in limbo for awhile. :/

In the short-to-medium-term, I'm inclined to pick my battles. I.e. I'd be ok with using more vanilla Varnish for most things, but then making sure the really high visibility stuff (Satlas, OLMo, etc.) looks super awesome.

@arnavic @jonborchardt thoughts?

**Comment by aaronsarnat** [comment]
Also, @arnavic in case you were wondering, this is why I've been dragging my feet a bit around the MUI to ai2-web migration effort. I agree MUI is an improvement over Antd, but I want to go deeper than that. I'm going to push as much as I can to get buy-in for (eventually) a new org-wide design system and brand refresh (look and feel but not logo), and thus AI2 Website redesign (and possible refactor).

I also would eventually like us to graduate from MUI and build our own custom Varnish component library from scratch... If we were ever able to afford the time to build that (and/or if we could start gradually incrementing towards that somehow) then I think what we'd und up with is a situation where we'd be able to have our cake and eat it, too. First principles of design and eng...

Varnish has a ton of potential.

It's also a dope name for a component library.
